pkg/stub/handler.go

```diff
@@ -77,6 +78,7 @@ func NewSamplesOperatorHandler(kubeconfig *restclient.Config) (*Handler, error)
 	h.templateFile = make(map[string]string)
 
 	h.mapsMutex = sync.Mutex{}
+	h.version = os.Getenv("RELEASE_VERSION")
```

Comment:
is caching this way really safe .... I had done it before with the prior version and found problems (can't remember the specifics off hand ... will have to go back and look.

Also, actually running through the flow probably would answer this question for me as well.

Reply:
You'd have a  better idea than I, but I can't see how it wouldn't be...  we only create the handler once, when the operator/controller starts up, right?  And the env variable value can't change unless the operator pod gets redeployed w/ a new env value (at which point we'd be restarting and pick up the new value).

Reply:
based on how my local testing goes, if need be, I'll revisit my PRs/commits in this space and see